### PR TITLE
Update Calendar Docs for new property on Calendar Card

### PR DESCRIPTION
**Pull request by zsarnett**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/frontend/pull/7135, https://github.com/home-assistant/frontend/pull/6982
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_lovelace/calendar.markdown [hunk 1]
```diff
@@ -16,6 +16,8 @@ The Calendar card displays your calendar entities in a month, day and list view.
 {% configuration_basic %}
 Title:
   description: The title displayed at the top of the card.
+Initial View:
+  description: The view that will load first on the card. Options are Month View, Day View or List View.
```
Comment (zsarnett) [inline source/_lovelace/calendar.markdown]:
```suggestion
  description: The view that will show first when the card is loaded on to the page. Options are Month View, Day View, or List View.
```

File: source/_lovelace/calendar.markdown [hunk 2]
```diff
@@ -35,6 +37,10 @@ title:
   required: false
   description: The title of the card.
   type: string
+initial_view:
+  required: false
+  description: The first view that will show on the card. Options are dayGridMonth, dayGridDay and listWeek.
```
Comment (zsarnett) [inline source/_lovelace/calendar.markdown]:
```suggestion
  description: The view that will show first when the card is loaded onto the page. Options are dayGridMonth, dayGridDay, and listWeek.
```

File: source/_lovelace/calendar.markdown [hunk 3]
```diff
@@ -16,6 +16,8 @@ The Calendar card displays your calendar entities in a month, day and list view.
 {% configuration_basic %}
 Title:
   description: The title displayed at the top of the card.
+Initial View:
+  description: The view that will show first when the card is loaded on to the page. Options are Month View, Day View, or List View.
```
Comment (zsarnett) [inline source/_lovelace/calendar.markdown]:
```suggestion
  description: The view that will show first when the card is loaded onto the page. Options are Month View, Day View, or List View.
```

File: source/_lovelace/calendar.markdown [hunk 4]
```diff
@@ -35,6 +37,10 @@ title:
   required: false
   description: The title of the card.
   type: string
+initial_view:
+  required: false
+  description: The view that will show first when the card is loaded onto the page. Options are dayGridMonth, dayGridDay, and listWeek.
```
Comment (klaasnicolaas) [inline source/_lovelace/calendar.markdown]:
```suggestion
  description: The view that will show first when the card is loaded onto the page. Options are `dayGridMonth`, `dayGridDay`, and `listWeek`.
```

File: source/_lovelace/calendar.markdown [hunk 5]
```diff
@@ -16,6 +16,8 @@ The Calendar card displays your calendar entities in a month, day and list view.
 {% configuration_basic %}
 Title:
   description: The title displayed at the top of the card.
+Initial View:
+  description: The view that will show first when the card is loaded onto the page. Options are Month View, Day View, or List View.
```
Comment (klaasnicolaas) [inline source/_lovelace/calendar.markdown]:
```suggestion
  description: The view that will show first when the card is loaded onto the page. Options are `Month View`, `Day View`, or `List View`.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_lovelace/calendar.markdown (hunk 4 above)
Comment (zsarnett) [reply source/_lovelace/calendar.markdown]:
```suggestion
  description: "The view that will show first when the card is loaded onto the page. Options are `dayGridMonth`, `dayGridDay`, and `listWeek`."
```

File: source/_lovelace/calendar.markdown (hunk 5 above)
Comment (zsarnett) [reply source/_lovelace/calendar.markdown]:
```suggestion
  description: "The view that will show first when the card is loaded onto the page. Options are `Month View`, `Day View`, or `List View`."
```

File: source/_lovelace/calendar.markdown (hunk 4 above)
Comment (zsarnett) [reply source/_lovelace/calendar.markdown]: Needed to add quotes around the text to do this

**Comment by zsarnett** [comment]
Parent PR merged

**Review by frenck — APPROVED** [review]
Thanks, @zsarnett 👍

**Pull request merged**